deploy/helm_charts/envs/mixer_prod.yaml

```diff
@@ -23,6 +23,18 @@ serviceGroups:
       resources:
         memoryRequest: "8G"
         memoryLimit: "8G"
+  - recon: 
+      urlPaths:
+        - "/entity/*"
+        - "/coordinate/*"
+        - "/id/*"
+      replicas:
+        default: 4
+        min: 4
+        max: 8
+      resources:
+        memoryRequest: "3G"
```

Comment:
@Spaceenter do you expect the recon API uses high memory? Or we can reduce the memory but increase the number of pods?

Reply:
As discussed offline with Bo, will keep this at 3G for now and observe what happens.